avro/src/main/java/com/fasterxml/jackson/dataformat/avro/annotations/AvroNamespace.java

```diff
@@ -0,0 +1,16 @@
+package com.fasterxml.jackson.dataformat.avro.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Annotation allows to override default Avro type namespace value.
+ * Default value is Java package name.
+ */
+@Target(ElementType.TYPE)
```

Comment:
To allow "annotation bundle" usage may also want to add `ElementType.ANNOTATION_TYPE`

Reply:
Right, good idea. 
ElementType.ANNOTATION_TYPE added to allow "annotation bundle" usage.